daemon/cmd/daemon_main.go

```diff
@@ -594,6 +594,10 @@ func InitGlobalFlags(logger *slog.Logger, cmd *cobra.Command, vp *viper.Viper) {
 	option.BindEnv(vp, option.ServiceNoBackendResponse)
 
 	flags.Int(option.TracePayloadlen, defaults.TracePayloadLen, "Length of payload to capture when tracing native packets.")
+
+	flags.String(option.PolicyDenyResponse, defaults.PolicyDenyResponse, "How to handle ipv4 egress traffic dropped by policy: either drop the packet (\"none\") or reject with an ICMP Destination Unreachable (\"icmp\")")
```

Comment:
```suggestion
	flags.String(option.PolicyDenyResponse, defaults.PolicyDenyResponse, "How to handle egress traffic dropped by network policy: either drop the packet (\"none\") or reject with an ICMP Destination Unreachable (\"icmp\")")
```

Otherwise we'll definitively forget this later :smile:. Maybe we also want to mention that it only targets pod traffic.

Of note - once all is settled, we'll also want an update for https://docs.cilium.io/en/latest/network/kubernetes/policy/

Reply:
Alright, updated this to "How to handle pod egress traffic dropped by network policy"

Also updated the docs in the last commit, although I put them in the Network Policy intro page: https://docs.cilium.io/en/latest/security/policy/intro/ (I feel like it fits a bit better there but happy to move them too)